docs/custom_port_forwards_and_links.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Custom port forward names and additional links for resources
+layout: docs
+---
+
+By default, port forward links for resources are displayed as `localhost:<port-number>` in Tilt, at the top of the screen. This may be confusing if you have multiple port forwards per resource, such as:
+
+<figure>
+    <img src="/assets/img/port-forwards.png" title="Port forwards">
```

Comment:
consider at least one image that shows more of the UI (with maybe the URLs circled/highlighted?) so user can orient themselves in the full UI. Right now this image gives no clue as to where in the UI it happens.